### DISCUSSION TITLE
alumni-aan-het-woord

### DISCUSSION THREAD

**Discussion by DphnZwp** [fdnd/docs.fdnd.nl#87] [Reflectie]
![spaarvarken](https://github.com/fdnd/docs.fdnd.nl/assets/69635977/96079675-d44b-44e9-8841-779e027d7b0f)
# Alumni - Daphne Zwuup
Heyy, ik ben Daphne (voor de mensen die me niet kennen). Ik had vorig jaar mijn diploma gehaald van FDND. Dus ik kom hier mijn verhaaltje vertellen over hoe ik de opleiding heb ervaren en wat ik nu doe.

## Wat vond je prettig?
Ik vond het heel erg prettig dat ik met meerdere opdrachtgevers had gewerkt in de opleiding. Door met meerdere opdrachtgevers te werken had ik heel veel websites om in mijn portfolio te doen. Ik had ook in die projecten met veel verschillende frameworks en libraries gewerkt om uitzoeken welke frontend technologieën het leukste is en om meer ervaring op te doen. Mijn portfolio en ervaring heeft me heel veel geholpen met het vinden van een baan. Heel veel reqruiters en twee bedrijven waren heel erg onder de indruk van mijn portfolio. Voor de rest vind ik Flexbox, Grid, UI states, Accessiblity en Performance belangrijke onderwerpen om te leren. Omdat dat mij ook veel heeft geholpen met mijn websites en kenningsmakingsgesprekken. Ik vond het ook altijd fijn om bijlessen te krijgen over bijvoorbeeld Eleventy en andere libraries.

## Wat heeft het je gebracht?
Innovative365 had voordat ik er was geen ontwerper en ze waren helemaal niet van plan om een Frontend Developer aan te nemen, omdat ze een CTO en Fullstack developer hebben naast hun backend developers. Maar toen ik ze mijn portfolio had laten zien en mijn ervaring had verteld, wouden ze me heel graag aannemen. Mijn werk draait voornamelijk om het Blazor-framework (C# framework) in combinatie met de componentenbibliotheek Radzen. Ik werk met verschillende talen, waaronder HTML, CSS, en een heeel klein beetje JavaScript (alleen voor één specifiek component). Sinds ik Blazor en Radzen gebruik om pagina’s te maken, heb ik geen JavaScript meer nodig (behalve voor die ene rich text component lol). In de website "Automate" van Innovative maak ik gebruik van de programmeertaal C# voor interacties en koppelingen, CSS voor de styling, en combineer ik Radzen- en HTML-elementen om de pagina’s te bouwen. Altijd wanneer ik klaar ben met een paginakunnen de backenders de backend koppelen aan mijn Frontend. Ik ben de enigste frontender en vrouw in het bedrijf en ik werk samen met 4 backenders, 1 fullstack developer en een CTO.

Ze hadden een bootstrap ontwerp voor hun Automate website en geen goede gebruikerservaring in sommige pagina's (bijv. heel veel onnodig drukken voor bepaalde acties). Daarom had ik een nieuwe huisstijl en nieuwe ontwerpen voor hun website gemaakt. Het is me gelukt om na 6 maanden hun oude website om te toveren naar mijn nieuwe ontwerp en om al de frontend te bouwen. Door mij zijn ze meer geïnteresserd geworden in de frontend willen ze dat mijn ontwerpen zelfs nog beter worden in de toekomst. Dus ik moet mijn ontwerpen nog meer verbeteren :). In de eerste 6 maanden was het doel die het bedrijf aan me heeft gegeven dat ik Blazor en Radzen leer. Dat doel is gelukt! Het volgende doel die ze aan me hebben gegeven is om de frontend te koppelen aan de backend data. Het gaat me denk ik zeker lukken om dit doel te behalen, omdat het me al gelukt is om twee pagina's te koppelen aan de backend.

## Wat kan er beter?
Soms waren er lessen niet gedaan en hadden we minder begeleiding in het tweede jaar, omdat docenten het druk hadden met de eerste jaars studenten. De PWA les (offline internet websites) was bijvoorbeeld niet gedaan of vergeten door de drukte met de eerstejaars. Ook kon ik een aantal keer niet met mijn begeleider Joost afspreken, omdat hij het te druk had of ziek was. Daarom zit ik meetal in mijn eentje met de opdrachtgever Iskandar wanneer ik bijvoorbeeld eerstejaars zit te begeleiden. Ik was ook een keertje helemaal alleen met de eerstejaars zonder Joost en Iskandar. Ik was hierdoor heel erg gestrest geraakt, omdat ik de brieving voor de eerstejaars helemaal in mijn eentje moest doen. Ik had de belangrijkste informatie verteld. Maar door de stress was ik helemaal vergeten om de huisstijl uit te leggen en om te laten zien welke websites ik heb gemaakt voor Coding The Curbs, zodat ze een idee hebben wat de nieuwe huisstijl van CTC is. Nandita was er wel, maar zij had geen verstand van CTC, dus ze kon me niet helpen herinnneren wat ik moest vertellen of laten zien.

## Wat vertel jij over je opleiding op een feestje?
Ik heb niet heel veel over FDND gezegd in feestjes. Ik vertel meestal wat ik doe in mijn Frontend Designer & Developer baan. Welke frameworks ik doe, programmeertalen, libraries etc. etc.

Ik hoop dat jullie hier iets aan hebben. Laat het me weten in de comments als jullie nog andere vragen hebben =).

**Comment by ju5tu5** [comment]
Opgenomen in het document
